pytket/docs/display.rst

```diff
@@ -10,21 +10,49 @@ Contains several functions for rendering interactive circuit diagrams.
         pip install pytket-offline-display
 
 .. automodule:: pytket.circuit.display
-    :members: render_circuit_jupyter, render_circuit_as_html, view_browser
```

Comment:
Are we removing these methods from the docs?

Reply:
I removed them from being listing explicitly since they are the same as those found in the class documentation that now gets listed, but I can add them back in if you think its helpful

Reply:
> I removed them from being listing explicitly since they are the same as those found in the class documentation that now gets listed, but I can add them back in if you think its helpful

Ah I see, no that's fine.